presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -84,6 +84,17 @@ For example:
      NULL        | NULL      | NULL                  | NULL        |      15.0
      (5 rows)
 
+If there is no table layout for provided SELECT (eg. because it can be determined that WHERE will filter all
```

Comment:
Table Layout is not a user-visible concept. Maybe you can talk about all partitions being pruned or something like that?